service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/OrderQuestionBlock.java

```diff
@@ -8,6 +8,9 @@
 @Getter
 @RequiredArgsConstructor
 public enum OrderQuestionBlock {
+    LINKED_TO_HEARING("hearingDetails", "Linked to hearing", OrderSection.HEARING_DETAILS,
```

Comment:
minor question, what is the point of the question field?

Reply:
no point, just readibility. Probably can be deleted in another Pr (just avoiding merge hell at this stage)